src/Symfony/Component/Runtime/Internal/autoload_runtime.template

```diff
@@ -0,0 +1,28 @@
+<?php
+
+// autoload_runtime.php @generated by Symfony Runtime
+
+use Symfony\Component\Runtime\SymfonyRuntime;
+
+if (true === (require_once __DIR__.'/autoload.php') || empty($_SERVER['SCRIPT_FILENAME'])) {
+    return;
+}
+
+if (!is_object($app = require $_SERVER['SCRIPT_FILENAME'])) {
+    throw new \TypeError(sprintf('Invalid return value: callable object expected, "%s" returned from "%s".', get_debug_type($app), $_SERVER['SCRIPT_FILENAME']));
+}
+
+$runtime = $_SERVER['APP_RUNTIME'] ?? SymfonyRuntime::class;
+$runtime = new $runtime(($_SERVER['APP_RUNTIME_OPTIONS'] ?? []) + ['project_dir' => %project_dir%]);
+
+[$app, $args] = $runtime
+    ->getResolver($app)
+    ->resolve();
+
+$app = $app(...$args);
+
+exit(
+    $runtime
+        ->getRunner($app)
+        ->__invoke() ?? 0
```

Comment:
this is wrong. `getRunner` is typed as returning a `callable`, not an object with a `__invoke` method (which is more restrictive).

Reply:
thanks for the review :)
getRunner is typed as returning a RunnerInterface|Closure, which both have the method:
https://github.com/symfony/symfony/pull/38465/files#diff-c6a00a8d0e38b89c945302378d5ce224ee4076c132a6bbe0062b875fb18ed0e7R33

Reply:
I just remove the Closure part and made `RunnerInterface` the only allowed return type.
The method in the interface is now `run()` instead of `__invoke()`.